### DISCUSSION TITLE
Examples for VoteRank algorithm

### DISCUSSION THREAD

**Discussion by dtekinoglu** [networkx/networkx#5461] [Ideas]
I see that there are no examples for VoteRank algorithm in the documentation. I think it might be useful to include some examples especially for non-trivial cases (such as multigraphs, existence of nodes with 0 votes etc.) If you too think that this can be useful, I can work on this issue.

**Comment by dtekinoglu** [comment]
@MridulS @dschult

**Reply by dschult** [reply]
Yes, that could be a useful place for examples in the doc_strings.
Also, you can "edit" a comment that you have already posted to e.g. add people's names, by clicking on the 3 dots at the top right and choosing "Edit" from the drop-down menu.   :}

**Reply by dtekinoglu** [reply]
Thank you @dschult , I will work on this.
